Projects/SEC/SEC04/SEC04-AWS100.md

```diff
@@ -0,0 +1,54 @@
+
+# SEC04-AWS100 — Create an IAM user
+
+## Cloud Service Provider
+
+* Amazon Web Services
+
+## Difficulty
+Level 100 (Introductory)
+
+## Estimated time:
+ 1 hour
+
+## Project's Author(s)
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
```

Comment:
- "Admin group" does not exist as such. Creating users for the very first time in IAM is not a walk in the park, as much as it seems like it to us now. Please explain this step in detail.
  - Create a new group called "Admins"
  - Add the `AdministratorAccess` managed policy to the new group
  - Add your new user to the "Admins" group

Reply:
Oops! You are right. Fixed this as well as subheaders under the Add user to Admin group header.